README.md

```diff
@@ -137,6 +141,33 @@ secrets:
           allow_sign_up: true
 ```
 
+#### Option 2: Reference Existing Secrets
+
+If you manage secrets externally (using External Secrets Operator, Sealed Secrets, manual provisioning, etc.), you can reference existing secrets instead:
+
+```yaml
+secrets:
+  # Reference existing Kubernetes secrets ids
+  existingControlPlaneSecret: "my-control-plane-secret"
+  existingHarmonySecret: "my-harmony-secret"
+```
+
+**Required keys for existing secrets:**
+
+The `existingControlPlaneSecret` must contain:
+- `dbUrl` - Database connection string
+- `cookiesSecret` - Cookie signing secret (>= 64 chars)
+- `oidc_providers` - OIDC configuration (see format in values.yaml)
+- `redisUrl` - Redis connection URL (auto-generated if using built-in Redis)
+
+The `existingHarmonySecret` must contain:
+- `ADAPTIVE_MODEL_REGISTRY` - S3 bucket URL for model registry
+- `ADAPTIVE_HARMONY__SHARED_DIRECTORY__URL` - S3 bucket URL for shared directory
+- `REDIS_URL` - Redis connection URL (auto-generated if using built-in Redis)
+- `HARMONY_SETTING_MODEL_REGISTRY_ROOT` - Same as ADAPTIVE_MODEL_REGISTRY
```

Comment:
Its a bit ugly that we don't have the same naming convention on both secrets I think we can change the deployments to just pick 1 naming convention and stick with it.

Also the duplicated redis url is not really user friendly. Maybe both components can read their data from 1 big secret to avoid duplication ? End-User don't really care if its used by control-plane or harmony.